src/app/screens/PeerlyScreen/index.tsx

```diff
@@ -1,15 +1,162 @@
 import React from 'react';
-import {SafeAreaView, StyleSheet, ScrollView, Text} from 'react-native';
+import {
+  View,
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  Text,
+  TextInput,
+  Image,
+  useWindowDimensions,
+  FlatList,
+} from 'react-native';
 
 import colors from '../../constant/colors';
+import AppreciationCard from './components/AppreciationCard';
+import {SceneMap, TabBar, TabView} from 'react-native-tab-view';
+import fonts from '../../constant/fonts';
+import FloatingGiveAppriciationButton from '../../components/button/floatingGiveAppriciationButton';
+import LeaderBoardCard from './components/LeaderBoardCard';
+
+const top10 = [
```

Comment:
Looks like this data will be fetched from api later on. Please remove this demo data.